driver/compify_dynlink.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
if you use `#!/bin/sh` instead, the script should still work and also work on BSD systems.